integrations/va-auth0/src/index.tsx

```diff
@@ -0,0 +1,251 @@
+        switch (action.action) {
+            case 'save.config':
+                const { api, environment } = context;
+                const spaceInstallation = environment.spaceInstallation;
+
+                const configurationBody = {
+                    ...spaceInstallation.configuration,
+                    client_id: element.state.client_id,
+                    client_secret: element.state.client_secret,
+                    issuer_base_url: element.state.issuer_base_url,
+                };
+                await api.integrations.updateIntegrationSpaceInstallation(
+                    spaceInstallation.integration,
+                    spaceInstallation.installation,
+                    spaceInstallation.space,
+                    {
+                        configuration: {
+                            ...configurationBody,
+                        },
+                    }
+                );
+                return element;
+        }
+    },
+    render: async (element, context) => {
+        const VACallbackURL = `${context.environment.spaceInstallation?.urls?.publicEndpoint}/visitor-auth/response`;
+        return (
+            <block>
+                <input
+                    label="Enter Client ID"
+                    hint="Enter Client ID of your Auth0 application"
+                    element={<textinput state="client_id" placeholder="Client ID" />}
+                />
+
+                <input
+                    label="Enter Your Auth0 Domain"
+                    hint="Enter Domain of your Auth0 instance"
+                    element={<textinput state="issuer_base_url" placeholder="Domain" />}
+                />
+
+                <input
+                    label="Enter Client Secret"
+                    hint="Enter Client Secret of your Auth0 application"
+                    element={<textinput state="client_secret" placeholder="Client Secret" />}
+                />
+
+                <input
+                    label=""
+                    hint=""
+                    element={
+                        <button
+                            style="primary"
+                            disabled={false}
+                            label="Save"
+                            tooltip="Save configuration"
+                            onPress={{
+                                action: 'save.config',
+                            }}
+                        />
+                    }
+                />
+                {!element.state.client_id ||
+                !element.state.client_secret ||
+                !element.state.issuer_base_url ? (
+                    <hint>
+                        <text style="bold">Enter values for the fields above and hit Save</text>
+                    </hint>
+                ) : null}
+                <divider size="medium" />
+                <text>Enter the following URL as an allowed callback URL in Auth0:</text>
+                <text>{VACallbackURL}</text>
```

Comment:
This will not look great. Why not use  a `codeblock` that also allows the user to copy the whole thing easily?